Error while checking build/O2Physics/o2 for 4ab4c583e205af310b768f9e63a3ba7877a064ad at 2026-04-28 11:25:
```
## sw/BUILD/O2Physics-latest/log
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGDQ/Tasks/mftMchMatcher.cxx:476:12: error: unused variable 'muonCollisionId' [-Werror=unused-variable]
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGDQ/Tasks/mftMchMatcher.cxx:463:26: error: unused variable 'muonPdgCode' [-Werror=unused-variable]
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/O2Physics/15990/4ab4c583e205af310b768f9e63a3ba7877a064ad/build_O2Physics_o2/pretty.html).

--- a/PWGDQ/Tasks/mftMchMatcher.cxx
+++ b/PWGDQ/Tasks/mftMchMatcher.cxx
@@ -460,8 +460,6 @@ struct mftMchMatcher {
                           TMFT const& mftTracks,
                           std::vector<std::pair<int64_t, int64_t>>& matchablePairs)
   {
-    static constexpr int muonPdgCode = 13;
-
     // outer loop on muon tracks
     for (const auto& muonTrack : muonTracks) {
       // only consider MCH standalone or MCH-MID matches
@@ -473,7 +471,6 @@ struct mftMchMatcher {
       if (!muonTrack.has_collision()) {
         continue;
       }
-      auto muonCollisionId = muonTrack.collisionId();
 
       // skip tracks that do not have an associated MC particle
       if (!muonTrack.has_mcParticle()) {

